docs/diagrams/junwei/AddSessionSequenceDiagram.puml

```diff
@@ -57,15 +57,17 @@ AddSessionCommand -> Model : hasSession(name, sessionToAdd)
 activate Model
 deactivate Model
 
+opt has name and does not have session
```

Comment:
Sorry, as mention on telegram, maybe putting the opt here is not a good idea? 

Since if there is any problem, we would have to return an command exception, and that means that the opt that it is successful will be everything else that happens on the bottom. Might be better to just have a opt for the failed instance and show the CommandException part.

![image](https://user-images.githubusercontent.com/24240408/114358225-44233800-9ba5-11eb-9d69-8a5d3a517b2a.png)